addons/account/models/account_move_line.py

```diff
@@ -3300,7 +3300,7 @@ def _get_aml_values(self, **kwargs):
             'reconcile_model_id': self.reconcile_model_id.id,
             'analytic_distribution': self.analytic_distribution,
             'tax_repartition_line_id': self.tax_repartition_line_id.id,
-            'tax_ids': [Command.set(self.tax_ids.ids)],
+            'tax_ids': [Command.set(self.tax_ids.ids)] + kwargs.pop('tax_ids', []),
```

Comment:
Genuine question, it's already orm commands right?

Reply:
ouepp, here is the unlink of a tax for example (important part being the kwargs)
<img width="513" height="98" alt="image" src="https://github.com/user-attachments/assets/c5d4ac4f-bbf6-4437-baf1-1ab7088e2cab" />